### ISSUE TITLE
don't highlight whitespace as red for zero weights

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#38]
It looks weird with feature imporances:

![2016-11-01 18 39 35](https://cloud.githubusercontent.com/assets/107893/19891578/f09fd7c2-a06a-11e6-83f6-385d49e245f0.png)

**Comment by lopuhin** [comment]
It also looks weird for non-text feature names, maybe we should highlight spaces only when we are sure that features are from text, or if the user ask for it.

**Comment by kmike** [comment]
@lopuhin makes sense.

**Comment by kmike** [comment]
I think that it is fixed by #61 for practical purposes.
